sql.txt

```diff
@@ -1,14 +1,29 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM [Customers] ORDER BY Country LIMIT 10 OFFSET 10;
```

Comment:
You don't have to put brackets, meaning "[]", around table names. They are for when you have special characters in the name of a column or other identifier.